backend/app/services/adapters/task_kinds/filters.py

```
@@ -0,0 +1,171 @@
+    try:
+        labels = task_crd.metadata.labels
+        if not labels:
+            return False
+
+        # Check for background task indicators
+        return (
+            labels.get("taskType") == "summary"
+            or labels.get("source") == "background_executor"
+            or labels.get("type") == "background"
+        )
+    except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)